libraries/AP_Scripting/docs/docs.lua

```diff
@@ -668,6 +668,10 @@ local RC_Channel_ud = {}
 ---@return number
 function RC_Channel_ud:norm_input_ignore_trim() end
 
+-- desc
+---@param param1 integer -- PWM value
+function RC_Channel_ud:set_override(param1) end
```

Comment:
```suggestion
---@param PWM integer
function RC_Channel_ud:set_override(PWM) end
```
As i mentioned on the other PR. This is what I have been doing.

Reply:
done